app/src/main/java/com/github/palFinderTeam/palfinder/tag/TagsRepository.kt

```diff
@@ -30,7 +30,7 @@ interface TagsRepository<T: Tag> {
 /**
  * Simple implementation if the tags should be readonly.
  */
-class NonEditableTags<T: Tag>(override val tags: Set<T>, override val allTags: Set<T>) : TagsRepository<T> {
+/*class NonEditableTags<T: Tag>(override val tags: Set<T>, override val allTags: Set<T>) : TagsRepository<T> {
```

Comment:
because they are not useful anymore (were made only for the purpose of the first presentation of the tags) and are therefore decreasing our code coverage. We could indeed even delete these 2 classes.